homeassistant/components/doorbird/__init__.py

```diff
@@ -24,11 +24,10 @@
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.exceptions import ConfigEntryNotReady
 import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.network import get_url
 from homeassistant.helpers.typing import ConfigType
-from homeassistant.util import dt as dt_util, slugify
 
 from .const import (
+    API_URL,
```

Comment:
This is no longer used in this file

Reply:
https://github.com/home-assistant/core/pull/98483/files#diff-ef4f5d2ab24ecd1c59aa71080b1c8400363d816ef8d7843abbbc754d1f1a71d4R216